docs/manager/graphql-reference/schema.graphql

```diff
@@ -360,6 +360,21 @@ type AuditLogSchema {
   Possible values of "AuditLogNode.status"
   """
   status_variants: [String]
+
+  """Possible values of audit log action types, Added in 25.8.0"""
+  action_type_variants: [ActionTypeVariant]
+}
+
+"""
+Possible values of audit log action types.
+Added in 25.8.0.
+"""
+type ActionTypeVariant {
+  """Entity type of the AuditLog"""
+  entity_type: String!
+
+  """Action types of the AuditLog"""
+  action_types: [String]!
```

Comment:
entity_type: A value that reveals what entity the service action is dealing with (ex: session, image, etc.)
action_types: Indicates the service actions that can be performed on the server (this value allows you to set which actions to collect audit logs for)